### ISSUE TITLE
Account Hide Profile from Searches and Online Status Not Saving

### ISSUE THREAD

**Issue by tigredanky** [ultimatemember/ultimatemember#239]
On the UM Account page for Privacy, toggling the "Hide my profile from directory" or "Hide my online status" is not saving my choice.

**Comment by tigredanky** [comment]
update... I've logged in as a lower member role and toggled the "hide from search" and it still went to "No", but when I check out the directory, it no longer displays me in the directory... so the toggle sounds like it's just not saving to show that it has been switched. thx. (note, this user role is not an Admin)

![screen shot 2017-02-16 at 4 36 22 pm](https://cloud.githubusercontent.com/assets/460065/23047676/2edda25e-f466-11e6-844e-e0a05c4072da.png)
![screen shot 2017-02-16 at 4 36 16 pm](https://cloud.githubusercontent.com/assets/460065/23047677/2ededa70-f466-11e6-8ec5-42397fc27451.png)

**Comment by champsupertramp** [comment]
Hi,

Please try the updated pre-release version 1.3.84:
https://github.com/ultimatemember/ultimatemember/releases

Regards,

**Comment by tigredanky** [comment]
I deactivated the existing UM plugin, uploaded the pre-release, but then all my UM settings got whacked and destroyed. Will wait for an official update to arrive. Thanks!

**Comment by champsupertramp** [comment]
Hi @tigredanky ,

Can you please provide a screenshot 'UM settings got whacked and destroyed' ?

Regards,

**Comment by tigredanky** [comment]
I reverted to my backup, and didn’t take any screen shots.

What happened was I tried to deactivate the UM plugin to delete it and then upload the pre-release, but it would deactivate. Then I uploaded the pre-release to my plugins folder directly and renamed it to ultimate-member and everything got wiped, so I reverted. Any documentation on how to safely update a pre-release version? I’ll try it again…
